README.md

```diff
@@ -9,6 +9,8 @@ The core repository is the starting point to engage in and learn about the
 > * [dotnet/cli](https://github.com/dotnet/cli) - for CLI tools and questions
 > * [dotnet/corefx](https://github.com/dotnet/corefx) - for API issues and questions
 > * [dotnet/coreclr](https://github.com/dotnet/coreclr) - for runtime issues
+> * [dotnet/docs](https://github.com/dotnet/docs) - for .NET Core documentation issues and questions
+> * [dotnet/sdk](https://github.com/dotnet/sdk) - for .NET Core SDK for Visual Studio issues and questions
```

Comment:
The SDK is used by more than just VS, specifically the CLI. So I think this should say:

> for .NET Core SDK issues and questions

Though that doesn't really say anything to those who don't already know what the SDK is, so maybe something like:

>  for .NET Core SDK (used by Visual Studio and CLI) issues and questions